q2_feature_classifier/classifier.py

```diff
@@ -133,20 +156,24 @@ def classify(reads: DNAIterator, classifier: Pipeline,
 
 
 _classify_parameters = {'chunk_size': Int, 'n_jobs': Int, 'pre_dispatch': Str,
-                        'confidence': Float}
+                        'confidence': Float, 'force_read_direction':
+                        Str % Choices(['forward', 'reverse'])}
 
 
 plugin.methods.register_function(
-    function=classify,
+    function=classify_sklearn,
     inputs={'reads': FeatureData[Sequence],
             'classifier': TaxonomicClassifier},
     parameters=_classify_parameters,
     outputs=[('classification', FeatureData[Taxonomy])],
-    name='Classify reads by taxon.',
+    name='Pre-fitted sklearn-based taxonomy classifier',
     description='Classify reads by taxon using a fitted classifier.',
     parameter_descriptions={'confidence': 'Confidence threshold for limiting '
                             'taxonomic depth. Negative value disables. '
-                            'Currently experimental. USE WITH CAUTION.'}
+                            'Currently experimental. USE WITH CAUTION.',
+                            'force_read_direction': 'Force reads to be treated'
```

Comment:
Thanks! See my followup comments [here](https://github.com/qiime2/q2-feature-classifier/pull/62#discussion_r110963462) and [here](https://github.com/qiime2/q2-feature-classifier/pull/62#discussion_r110963736).